### PR TITLE
feat: [PIE-4266]: Email Step

### PR DESCRIPTION
**Pull request by Eric761**
### Summary

Email Step

#### Screenshots

Screenshot 2022-07-11 at 7 00 18 AM

https://user-images.githubusercontent.com/73639897/178178342-56cbadf8-fdf3-4026-a762-8bd42ca8a12e.mov

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by Eric761** [comment]
retrigger typecheck
retrigger titlecheck

**Comment by Eric761** [comment]
retrigger typecheck
retrigger titlecheck

**Comment by Eric761** [comment]
retrigger typecheck

**Comment by Eric761** [comment]
retrigger titlecheck

**Comment by Eric761** [comment]
retrigger titlecheck

**Comment by Eric761** [comment]
retrigger coverage

**Comment by Eric761** [comment]
retrigger eslint

**Comment by Eric761** [comment]
retrigger coverage

**Comment by Eric761** [comment]
retrigger coverage

**Comment by Eric761** [comment]
retrigger jest

**Comment by Eric761** [comment]
trigger cypress

File: src/modules/70-pipeline/strings/strings.en.yaml [hunk 1]
```diff
@@ -81,6 +81,9 @@ utilitiesStep:
   url: Enter the URL
   requestBody: Enter the message body
   assertion: Enter Assertion
+  to: Enter recepient email
```
Comment (rameshx) [inline src/modules/70-pipeline/strings/strings.en.yaml:84]:
```suggestion
  to: Enter recipient email
```

**Review by vkbansal — APPROVED** [review]

**Review by rameshx — DISMISSED** [review]

**Review by vkbansal — APPROVED** [review]

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Failed_** 
 


            
 #### 122 Issues 
* Code issue 122 Code issues 

 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/9931)

**Comment by Eric761** [comment]
retrigger coverage

**Review by vikrant-harness — APPROVED** [review]

**Review by vikrant-harness — APPROVED** [review]

**Review by vkbansal-harness — APPROVED** [review]

**Pull request merged**